manpages/lumina-mediaplayer.8

```diff
@@ -0,0 +1,27 @@
+.Dd October 20, 2017
+.Dt LUMINA-MEDIAPLAYER 8
+.Os 1.3.3
+
+.Sh NAME
+.Nm lumina-mediaplayer
+.Nd is a simple graphical utility that plays various media files. It also has hooks to stream online radio services.
+
+.Sh SYNOPSIS
+.Nm
+.Op <File Name>
+
+.Sh DESCRIPTION
```

Comment:
"that allows the user to do several things" is unnecessary -> ".Nm opens a graphical interface. The user can sync..."